src/d3a/d3a_core/util.py

```diff
@@ -574,3 +574,10 @@ def return_ordered_dict(function):
     def wrapper(*args, **kwargs):
         return OrderedDict(sorted(function(*args, **kwargs).items()))
     return wrapper
+
+
+def get_simulation_queue():
```

Comment:
Only because I was originally confused that this was an function that fetched the rq.Queue object, I would recommend to rename it to `get_simulation_queue_name`